### PR TITLE
feat(exchange): move action button to the bottom of the card

### LINKED ISSUE #1697: feat(exchange): move action button to the bottom of the card
- [ ] move action button to the bottom of the exchange card

### LINKED ISSUE #1744: bug(bookmark): invisible button on exchange with connected
- [x] new regression on the bookmark button makes is invisible :(

### PR DESCRIPTION
**Pull request by JNSPK**
<p align =center>${\color{lightgreen}FIX}$ </p></p><p align=center></p><p align =right>fix #1697 and fix #1744 </p>

### CODE REVIEW

File: apps/www/widgets/exchanges/card/index.tsx [hunk 1]
```diff
@@ -185,6 +163,26 @@ function Idle() {
...
+              href={alreadyPopulated ? "" : `/@~/exchanges/${exchange.id}/edit`}
+              onClick={on_edit_click}
+            >
+              <Pen className="h-4" />
+            </Link>
+            {showInfoBox && (
+              <InfoBox message="Les échanges comprenant déjà des participants ne peuvent pas être édités" />
+            )}
+            <Exchange_Delete_Button />
+          </>
```
Comment (douglasduteil) [inline apps/www/widgets/exchanges/card/index.tsx:184]: *suggestion:* extract this to a separate component

File: packages/@1/modules/exchange/ui/src/Card/Card_Idle.tsx [hunk 2]
```diff
@@ -77,7 +78,13 @@ export function Card_Idle({ children }: PropsWithChildren) {
             ...
           </Card.Footer.Center.Renderer>
         </div>
-        <div className="col-start-6 justify-self-end">
+        <div
+          className={
+            is_yours
+              ? "col-start-6 columns-3 items-center gap-4 justify-self-end"
+              : "col-start-6 justify-self-end"
+          }
```
Comment (douglasduteil) [inline packages/@1/modules/exchange/ui/src/Card/Card_Idle.tsx:86]: *suggestion:* add a slot to deal with the variant

**Review by douglasduteil — COMMENTED** [review]
Not bad brooo

File: packages/@1/modules/exchange/ui/src/Card/Card_Idle.tsx (hunk 2 above)
Comment (JNSPK) [reply packages/@1/modules/exchange/ui/src/Card/Card_Idle.tsx]: > _suggestion:_ add a slot to deal with the variant

✅

File: apps/www/widgets/exchanges/card/index.tsx (hunk 1 above)
Comment (JNSPK) [reply apps/www/widgets/exchanges/card/index.tsx:184]: Can we do a refacto issue for that ?
I don't see why how and where to do that 😅

Comment (JNSPK) [reply apps/www/widgets/exchanges/card/index.tsx]: Done ✅

**Pull request merged**